tests/unittest/_torch/modeling/test_qwen_vl_processor_defaults_fix.py

```
@@ -0,0 +1,258 @@
+    saved = copy.deepcopy(kwargs_cls._defaults)
+    try:
+        try:
+            proc._get_num_multimodal_tokens(
+                image_sizes=[(2, 2)],
+                video_sizes=[(2, 2, 2)],
+                video_grid_thw="leak",
+            )
+        except Exception:
+            # The upstream method may raise on stub input; that's fine — the
+            # mutation we care about happens before the raise.
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)